app/src/main/res/values/colors.xml

```diff
@@ -133,7 +133,7 @@
     <color name="preference_section_header_dark_theme">@color/accent_high_contrast_dark_theme</color>
 
     <color name="mozac_widget_favicon_background_dark_theme">@color/photonDarkGrey50</color>
-    <color name="mozac_widget_favicon_border_dark_theme">@color/photonDarkGrey10</color>
+    <color name="mozac_widget_favicon_border_dark_theme">@color/dark_grey_90</color>
```

Comment:
We probably don't want to change this variable since it will affect all favicon that uses this variable unless this is what we want. I think you can change this to red to see what it would affect as an investigation and maybe see if we wanted to do this for all favicon. I would assume no naively.

What I would propose here is to create a new variable for the top sites background similar to how we have  top_sites_favicon_padding. You will need to define `top_sites_border` and presumably the `top_sites_border_light_theme` and `top_sites_border_dark_theme`.